src/Infrastructure/IfcViewerAPIExtended.js

```diff
@@ -84,7 +85,7 @@ export class IfcViewerAPIExtended extends IfcViewerAPI {
         await this.pickIfcItemsByID(modelID, this._selectedExpressIds, focusSelection, true)
         this.highlighter.setHighlighted(this.IFC.selector.selection.meshes)
       } catch (e) {
-        console.error(e)
+        debug().log('IfcViewerAPIExtended#setSelection$onError: ', e)
```

Comment:
you can actually use `debug().error()` for cases like this.. all of the methods from console are still there, or mocked in tests